src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -2550,6 +2570,28 @@ void FlexDRWorker::route_queue_init_queue(queue<RouteQueueEntry>& rerouteQueue)
       // no need to clear the net because route objs are not pushed to the net
       // (See FlexDRWorker::initNet)
     }
+  } else if (getRipupMode() == 2) {
```

Comment:
In the future it would be nice to replace these numbers with an enum of meaningful names.